tools/Custom/EventHelper.cs

```diff
@@ -0,0 +1,93 @@
+using System;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Graph.PowerShell.Runtime;
+
+namespace Microsoft.Graph.PowerShell
```

Comment:
I would make this an EventFactory class and move Print into an EventExtensions class.  The Helper suffix just isn't very helpful.